customer-wallet/src/containers/Transactions/index.js

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react'
+import { shape, bool, any, object, func, number } from 'prop-types'
+import { Link } from 'react-router-dom'
+// redux
+import { connect } from 'react-redux'
+import { compose } from 'redux'
+// helpers
+import get from 'lodash/get'
+// hoc
+import withCustomerTransactions from '../Hoc/withCustomerTransactions'
+import withFxRates from '../Hoc/withFxRates'
+// components
+import { Card, CardHeader, Avatar, withStyles } from '@material-ui/core';
+import TransactionInfo from './TransactionInfo'
+// icons
+import WalletIcon from '@material-ui/icons/AccountBalanceWallet'
+import ErrorIcon from '@material-ui/icons/ErrorOutline'
+import LoadingIcon from '@material-ui/icons/Sync'
+
+const styles = {
```

Comment:
@fazo96 if there are these styles kind of common and being reused, would it be nicer to refactor them out into a separate file. No need to do it now, just noting that we could think about it for future changes.